src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -187,7 +187,7 @@ def load_arguments(self, _):
         c.argument('yes', help="Do not prompt for preview.")
         c.argument('skip_features', help="Import only key values and exclude all feature flags. By default, all feature flags will be imported from file or appconfig. Not applicable for appservice.", arg_type=get_three_state_flag())
         c.argument('content_type', help='Content type of all imported items.')
-        c.argument('import_mode', arg_type=get_enum_type([ImportMode.ALL, ImportMode.IGNORE_MATCH]), help='If import mode is "ignore-match", source key-values that already exist at the destination will not be overwritten. Import mode "all" writes all key-values to the destination regardless of whether they exist or not.')
+        c.argument('import_mode', arg_type=get_enum_type([ImportMode.ALL, ImportMode.IGNORE_MATCH]), help='If import mode is "ignore-match", only key-values that do not already exist or whose key, label, value, content-type or tags are different from what is in the destination will be written. Import mode "all" writes all key-values to the destination regardless of whether they exist or not.')
```

Comment:
Thinking about it a bit more, do you think this would be clearer?

```
only source key-values that do not already exist or whose value, content-type or tags are different from that of an existing key-value with the same key and label in the destination, will be written
```

Thoughts on this?

Reply:
Yeah this sounds good. I have updated it